sdk/clientcore/pom.xml

```diff
@@ -14,7 +14,6 @@
     <module>http-okhttp3</module>
     <module>optional-dependency-tests</module>
     <module>http-stress</module>
-    <module>annotation-processor</module>
     <module>annotation-processor-test</module>
```

Comment:
Shouldn't `annotation-processor-test` move as well?

Reply:
We decided to keep this in clientcore based on our meeting discussion yesterday.